website/app/components/doc/page/header/index.hbs

```diff
@@ -21,24 +22,28 @@
           discover the search results.</div>
         <button
           type="button"
-          class="doc-page-header__search-button"
+          class="doc-page-header__desktop-icon-item"
           data-doc-algolia-search-autocomplete-secondary-trigger
           aria-describedby="search-button"
+          aria-label="Search"
         >
           <Hds::Icon @name="search" />
```

Comment:
**todo:** The size was only changed in CSS, so the stroke weight isn't displayed appropriately. The component needs to be updated as well. 

```suggestion
          <Hds::Icon @name="search" @size="24" />
```